app/Console/Kernel.php

```
@@ -0,0 +1,69 @@
+            $commandName = $input->getFirstArgument();
+
+            if ($commandName && str_contains($commandName, ':')) {
+                return $this->tryComponentDelegation($commandName, $input, $output);
+            }
+
+            // Re-throw if not a component command
+            throw $e;
+        }
+    }
+
+    private function tryComponentDelegation(string $commandName, InputInterface $input, OutputInterface $output): int
```

PHPMD (2.15.0)

Avoid unused parameters such as '$output'. (Unused Code Rules)

(UnusedFormalParameter)